resources/content/schema/base/snapshot.json

```diff
@@ -4,10 +4,6 @@
             "required": true,
             "nullable": false
         },
-        "managedSnapshotUUID": {
```

Comment:
And now snapshot.uuid and "managedVolumeUUID" in snapshot are needed for restore volumes.